### PR TITLE
create services for ticket summary and sentiment

### PR DESCRIPTION
**Pull request by darshanitaliya7110**
### What this does
Connect AI -> Implement services to generate ticket summary & contact sentiments

### Implementation
create services for ticket summary and sentiment

### CODE REVIEW

File: lib/openAi.ts [hunk 1]
```diff
@@ -8,3 +8,16 @@ if (!openAiAPIStr) {
...
+
+export const chatWithOpenAi = async (content: string) => {
+  const chatCompletion = await openAi.chat.completions.create({
+    messages: [
+      {
+        role: 'user',
+        content: `${content}`,
+      },
+    ],
+    model: 'gpt-3.5-turbo',
```
Comment (vatsal-pixer) [inline lib/openAi.ts:20]: Move this to .env with variable name `CONNECT_AI_MODEL=""`

File: services/serverSide/ticketSummary.ts [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  );
+
+  return { summary: summary.choices[0].message.content };
+};
+
+// Get ticket sentiment based on message data
+export const getTicketSentiment = async (ticketId: string) => {
+  const ticketContent = await getContent(ticketId);
+
+  const sentiment = await chatWithOpenAi(
```
Comment (vatsal-pixer) [inline services/serverSide/ticketSummary.ts:63]: Sentiment should of Contact (person whom contacting due to having trouble)

File: services/serverSide/ticketSummary.ts [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+// Get ticket summary based on message data
+export const getTicketSummary = async (ticketId: string) => {
+  const ticketContent = await getContent(ticketId);
+
+  const summary = await chatWithOpenAi(
+    `${ticketContent} \n\n\n Now give summary for above content in 1-2 lines.`,
+  );
+
+  return { summary: summary.choices[0].message.content };
```
Comment (vatsal-pixer) [inline services/serverSide/ticketSummary.ts:56]: Just directly return `summary.choices[0].message.content` don't return in object.

File: services/serverSide/ticketSummary.ts [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+// Get ticket sentiment based on message data
+export const getTicketSentiment = async (ticketId: string) => {
+  const ticketContent = await getContent(ticketId);
+
+  const sentiment = await chatWithOpenAi(
+    `${ticketContent} \n\n\n Now give sentiment for above content in 1 line`,
+  );
+
+  return { sentiment: sentiment.choices[0].message.content };
```
Comment (vatsal-pixer) [inline services/serverSide/ticketSummary.ts:67]: Just directly return `sentiment.choices[0].message.content` don't return in object.

File: services/serverSide/ticketSummary.ts (hunk 2 above)
Comment (darshanitaliya7110) [reply services/serverSide/ticketSummary.ts]: Done

File: services/serverSide/ticketSummary.ts (hunk 3 above)
Comment (darshanitaliya7110) [reply services/serverSide/ticketSummary.ts]: Done

**Pull request merged**